src/main/java/com/uwetrottmann/tmdb2/TmdbInterceptor.java

```diff
@@ -39,10 +70,95 @@ public static Response handleIntercept(Chain chain, String apiKey) throws IOExce
         HttpUrl.Builder urlBuilder = request.url().newBuilder();
         urlBuilder.setEncodedQueryParameter(Tmdb.PARAM_API_KEY, apiKey);
 
+        if (request.url().pathSegments().get(0).equals("account")) {
```

Comment:
It's not necessarily a bad idea to do preemptive authentication in an interceptor, when you know beforehand that the call requires it. An Authenticator should still be used for the cases where the could session token ends up being invalid (e.g. for attempting to refresh the token like you do in thetvdb).